src/link/SpirV.zig

```diff
@@ -307,6 +307,7 @@ fn writeMemoryModel(spv: *SpvModule, target: std.Target) !void {
 
     const addressing_model = switch (target.os.tag) {
         .opencl => switch (target.cpu.arch) {
+            .spirv => spec.AddressingModel.Logical,
```

Comment:
As I'm familiarizing myself with this stuff more, I'm becoming a lot less confident in this particular change. Doesn't *seem* like OpenCL can have logical memory layout?

Reply:
[correct](https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_Env.html#validation-rules)

Reply:
Thanks; removed this part.